nutridine/src/App.js

```diff
@@ -20,8 +20,11 @@ function App() {
 
   return (
     <Layout>
-      <MacroPage />
-      {/* <FoodCard /> */}
+      <Routes>
+        <Route path="/" element={<div>Home Page</div>} />
+        <Route path="/restaurants" element={<div>Restaurants Page</div>} />
+        <Route path="/macro" element={<MacroPage />} />
+      </Routes>
     </Layout>
```

Comment:
This is totally not on you, but would you mind adding this beneath the `</Routes>` please? We need to be rendering this as long as we use NutritionIX, or they'll ban us from using the API.

`<Image maxH="50px" src={nutritionIx_credit} />`

Reply:
Added